benchexec/tools/ultimate.py

```diff
@@ -13,13 +13,11 @@
 import re
 import shutil
 import subprocess
-import sys
 
 import benchexec.result as result
 import benchexec.tools.template
 import benchexec.util as util
 from benchexec import BenchExecException
```

Comment:
For the cases where this exception is currently used, `ToolNotFoundException` seems a good replacement.